test/TEST-24-CRYPTSETUP/test.sh

```diff
@@ -39,6 +41,86 @@ check_result_qemu() {
     return $ret
 }
 
+setup_pkcs11_token() {
+    local P11_MODULE_CONFIGS_DIR P11_MODULE_DIR SOFTHSM_MODULE
+
+    export SOFTHSM2_CONF="/tmp/softhsm2.conf"
+    mkdir -p "$initdir/var/lib/softhsm/tokens/"
+    cat >> ${SOFTHSM2_CONF} <<EOF
```

Comment:
We generally got rid of the space between redirection op and the argument. Below too.

I think this should be `>`, to avoid being confused by earlier runs and such.